src/components/Reserve.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-console */
+import { useState } from 'react';
+import { useParams } from 'react-router';
+import {
+  Container, Row, Col, Form, Button, Alert,
+} from 'react-bootstrap';
+import { useDispatch, useSelector } from 'react-redux';
+import { addReservation } from '../redux/reservations/reservations';
+import '../styles/Reserve.scss';
+
+const Reserve = () => {
+  const dispatch = useDispatch();
+  const { vehicle_Id: urlVehicleId } = useParams();
+  console.log('here', urlVehicleId);
```

Comment:
- [ ] Kindly remove this console statement in order to improve the cleanliness of your app.